debian/patches/retain-netplan-world-readable.patch

```diff
@@ -8,7 +8,7 @@ Last-Update: 2023-01-09
 ---
 This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
 --- a/cloudinit/features.py
-+++ b/cloudinit/features.py
++++ cloud-init/cloudinit/features.py
```

Comment:
Shouldn't this still be an `ab` diff?

```suggestion
+++ b/cloudinit/features.py
```

Reply:
I think both techincally work, but yes, we shouldn't change it mid-stream. Do you want to go ahead and make this change since Chad will be out for a bit and then I can approve and merge.